win32ss/user/ntuser/win32.h

```diff
@@ -135,7 +135,6 @@ typedef struct _THREADINFO
     WCHAR               wchInjected;
     UINT                cWindows;
     UINT                cVisWindows;
-#ifndef __cplusplus /// FIXME!
```

Comment:
What's happening here with C++?

Reply:
The `CLIENTTHREADINFO` structure is defined in `win32ss/include/ntuser.h` that had been skipped in the C++ compilation unit.